### ISSUE TITLE
crash on launch with architectury

### ISSUE THREAD

**Issue by yesn0t** [Pepe20129/difficultytweaker#7]
reproduced with :
* difficutlytweaker 0.2.4+1.18
* architectury 3.6.15 or 3.5.12
* fabric-api 0.46.3 or 0.46.1
* fabric-loader 0.13.0 or 0.12.12
* minecraft 1.18.1
* java 17

```
InvalidInjectionException: InjectionPoint(Shift)[@At("INVOKE")] 
    on net/minecraft/class_2910::checkPhantomSpawn with priority 1000 cannot inject 
    into net/minecraft/class_2910::method_6445(Lnet/minecraft/class_3218;ZZ)I 
    merged by io.github.pepe20129.difficultytweaker.mixin.PhantomSpawnerMixin with priority 1000 
          [PREINJECT Applicator Phase -> architectury.mixins.json:MixinPhantomSpawner -> Prepare Injections -> ...
```

<details><summary> full crash logs here </summary>

```
Description: Exception in server tick loop

java.lang.RuntimeException: Mixin transformation of net.minecraft.class_2910 failed
	at Not Enough Crashes deobfuscated stack trace.(1.18.1+build.9)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:252)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at net.minecraft.server.MinecraftServer.createWorlds(MinecraftServer:353)
	at net.minecraft.server.MinecraftServer.loadWorld(MinecraftServer:327)
	at net.minecraft.server.integrated.IntegratedServer.setupServer(IntegratedServer:72)
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer:657)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer:270)
	at java.lang.Thread.run(Thread.java:833)
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247)
	... 9 more
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [architectury.mixins.json:MixinPhantomSpawner] from phase [DEFAULT] in config [architectury.mixins.json] from mod [architectury] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:638)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:589)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:379)
	... 12 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: InjectionPoint(Shift)[@At("INVOKE")] on net/minecraft/class_2910::checkPhantomSpawn with priority 1000 cannot inject into net/minecraft/class_2910::method_6445(Lnet/minecraft/class_3218;ZZ)I merged by io.github.pepe20129.difficultytweaker.mixin.PhantomSpawnerMixin with priority 1000 [PREINJECT Applicator Phase -> architectury.mixins.json:MixinPhantomSpawner -> Prepare Injections ->  -> handler$zdk000$checkPhantomSpawn(Lnet/minecraft/class_3218;ZZLorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;Ljava/util/Random;ILjava/util/Iterator;Lnet/minecraft/class_1657;Lnet/minecraft/class_2338;Lnet/minecraft/class_1266;Lnet/minecraft/class_3442;IILnet/minecraft/class_2338;Lnet/minecraft/class_2680;Lnet/minecraft/class_3610;Lnet/minecraft/class_1315;IILnet/minecraft/class_1593;)V -> Prepare]
	at Not Enough Crashes deobfuscated stack trace.(1.18.1+build.9)
	at org.spongepowered.asm.mixin.injection.code.Injector.findTargetNodes(Injector.java:305)
	at org.spongepowered.asm.mixin.injection.code.Injector.find(Injector.java:240)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.prepare(InjectionInfo.java:421)
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1338)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:1043)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:393)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	... 12 more

```
</details>

let me know if i should report this to architectury instead, i'm not sure x)

**Comment by Pepe20129** [comment]
This is 100% my fault. I'll try to fix it, but it could take some time since I'm pretty bad at mixins.

**Comment by yesn0t** [comment]
all good, this mod otherwise ticks pretty much all the right boxes, thanks for making it :D

**Comment by Pepe20129** [comment]
Fixed in 0.2.5
